js-exercises/cache-function/cacheFunction.js

```diff
@@ -0,0 +1,15 @@
+const cacheFunction = (fn) => {
+  let cache = {}
+  return (...args) => {
+    let n = args[0]
+    if (n in cache) {
+      return cache[n]
+    } else {
```

Comment:
A better approach would be to just return `cache[n]` after conditionally inserting the value.